src/main/java/de/javakaffee/kryoserializers/guava/ArrayListMultimapSerializer.java

```diff
@@ -0,0 +1,45 @@
+package de.javakaffee.kryoserializers.guava;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.Serializer;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+
+import com.google.common.collect.ArrayListMultimap;
+
+/**
+ * A kryo {@link Serializer} for guava-libraries {@link ArrayListMultimap}.
+ * This does not yet support {@link Kryo#copy(java.lang.Object)}.
+ */
+public class ArrayListMultimapSerializer extends MultimapSerializerBase<Object, Object, ArrayListMultimap<Object, Object>> {
+
+    private static final boolean DOES_NOT_ACCEPT_NULL = false;
+
+    private static final boolean IMMUTABLE = false;
+
+    public ArrayListMultimapSerializer() {
+        super(DOES_NOT_ACCEPT_NULL, IMMUTABLE);
```

Comment:
I'm copying the existing classes here but I'm confused: in `ImmutableMapSerializer`, etc., this `DOES_NOT_ACCEPT_NULL` is passed for `Serializer`'s constructor's `acceptsNull` argument, which seems a bit backward? The names and values *may* need flipping, but the integration tests pass. :confused: